src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -20,6 +20,10 @@
 
 # VM
 
+def empty_on_404(ex):
+    if ex.status_code == 404:
```

Comment:
`status_code` may not exist on `ex`.
Would have to check that `isinstance(ex, what_youre_expecting)`